app/models/like.rb

```diff
@@ -0,0 +1,16 @@
+class Like < ActiveRecord::Base
+  belongs_to :author, class_name: 'User'
+  belongs_to :posts
```

Comment:
- [ ] Great work implementing the change requested by the other reviewer once again. There is a little change to be made here. The `Like` belongs to a single `Post` and not multiple posts. Kindly show this when defining the relationship between them. `belongs_to :post`. 👍